Jenkinsfile

```
@@ -0,0 +1,41 @@
+def VENV = 'wazo-benchmark-venv'
+
+pipeline {
+  agent {
+    label 'built-in'
+  }
+  environment {
+    MAIL_RECIPIENTS = 'dev+tests-reports@wazo.community'
+  }
+
+  options {
+    disableConcurrentBuilds()
+    timeout(time: 10, unit: 'MINUTES')
+    buildDiscarder(logRotator(numToKeepStr: '20'))
+  }
+  stages {
+    stage ("Benchmark"){
+      steps {
+        sh """
+          set -xe
+          python3.9 -m venv --clear $VENV
+          . $VENV/bin/activate
+          pip install wheel
+
+          pip install -r requirements.txt
+
+          fab -H root@wazo-benchmark.lan.wazo.io reset-server
+          pytest
```

Could be nice to add junit reporting output(https://docs.pytest.org/en/7.1.x/how-to/output.html#creating-junitxml-format-files).
```suggestion
          pytest --junitxml=./$(date -I)-junit.xml
```

---

https://plugins.jenkins.io/junit/
Not sure if we already integrate test reports in jenkins with this junit plugin or something else.

---

A nice to have obviously, could go into future improvements